hack/run-functests.sh

```diff
@@ -25,9 +27,20 @@ echo "Focusing on $FOCUS"
 
 export SUITES_PATH=cnf-tests/bin
 
-if ! cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit /tmp/artifacts/ -report /tmp/artifacts/; then
-  failed=true
-  failures+=( "Tier 2 tests for $FEATURES" )
+if [ $CNF_TESTS_IMAGE != "" ]; then
+  cp -f $KUBECONFIG _cache/kubeconfig
```

Comment:
do we need to add it (_cache) to .gitignore?

Reply:
It's already added.